README.md

```diff
@@ -22,6 +22,7 @@ sequential (nonprogressive) JPEGs due to faster decompression speeds they offer.
     *   On Ubuntu, do `apt-get install libpng-dev`.
     *   On Fedora, do `dnf install libpng-devel`. 
     *   On Arch Linux, do `pacman -S libpng`.
+    *   On Alpine Linux, do `apk add libpng-dev`.
```

Comment:
Do you think that reminding people about `lbu commit`, if appropriate for their install, would be useful here?